galpy/potential_src/SpiralArmsPotential.py

```diff
@@ -33,42 +33,60 @@ def func_wrapper(self, R, z, phi, t):
 
 
 class SpiralArmsPotential(Potential):
-    """Class that implements the spiral arms potential from Cox and Gomez (2002). Should be used to modulate an existing
-    potential. Left handed coordinate system. Unhandled division by zero errors if R == 0.
+    """Class that implements the spiral arms potential from (`Cox and Gomez 2002 <https://arxiv.org/abs/astro-ph/0207635>`__). Should be used to modulate an existing
+    potential.
     
     .. math::
     
-        \\Phi(r, \\phi, z) = -4 \\pi GH \\rho_0 exp(-\\frac{r-r_0}{R_s}) \\sum(\\frac{C_n}{(K_n D_n} cos(n \\gamma) sech(\\frac{K_n z}{B_n})^B_n)
+        \\Phi(r, \\phi, z) = -4 \\pi GH \\rho_0 exp \\left( -\\frac{r-r_0}{R_s} \\right) \\sum{\\frac{C_n}{K_n D_n} cos(n \\gamma) sech \\left( \\frac{K_n z}{B_n} \\right)^{B_n}}
 
+    where
+
+    .. math::
+        K_n &= \\frac{n N}{r sin(\\alpha)} \\\\
+        B_n &= K_n H (1 + 0.4 K_n H) \\\\
+        D_n &= \\frac{1 + K_n H + 0.3 (K_n H)^2}{1 + 0.3 K_n H} \\\\
+
+    From (`Cox and Gomez 2002 <https://arxiv.org/abs/astro-ph/0207635>`__):
```

Comment:
Not entirely sure how this looks when rendered, but do you also mention the default case C=1?